src/Redux/actionCreators/ServiceActions.js

```diff
@@ -117,7 +117,8 @@ export const downloadAuthToken = (serviceId, groupId, publicKey, orgId) => async
     );
 
     const jsonToDownload = {
-      tokeToMakeFreeCall: data.token_to_make_free_call,
+      email: userId,
```

Comment:
I think this shoudl be ok , but good to confirm with @vsbogd as well since we have a new attribute in json dowloaded

Reply:
He already reviewed it and also suggested the typo fix.
But yeah, could you please approve it @vsbogd?

Reply:
This change looks good to me :+1: